net-dns/dnsdist/files/dnsdist.conf

```diff
@@ -0,0 +1,2 @@
+-- This is a dnsdist configuration file.
+-- Please refer to http://dnsdist.org/README/ for more information.
```

Comment:
I dare say that's not very useful template. I don't see much of a point in installing a template that only gives URL that can be found in README or alike.